app/uk/gov/hmrc/merchandiseinbaggage/views/ViewUtils.scala

```diff
@@ -41,4 +42,21 @@ object ViewUtils {
 
   lazy val exportCountries: List[Country] =
     CountryService.getAllCountries.filterNot(_.code == "GB")
+
+  def proofOfOriginNeeded(declaration: Declaration): Boolean = {
+    def calcAmount(maybeTotalCalculationResult: Option[TotalCalculationResult]): Long =
+      maybeTotalCalculationResult.fold(0L) {
+        _.calculationResults.calculationResults
+          .filter(_.goods.producedInEu == YesNoDontKnow.Yes)
+          .map(_.gbpAmount.value)
+          .sum
+      }
+
+    if (declaration.declarationType == Import) {
+      (calcAmount(declaration.maybeTotalCalculationResult) + declaration.amendments
+        .filter(amendment => List(Some(Paid), Some(NotRequired)).contains(amendment.paymentStatus))
+        .map(x => calcAmount(x.maybeTotalCalculationResult))
+        .sum) > 100000L
+    } else false
+  }
```

Comment:
we can move this into `object DeclarationView` but that's fine for now, i have approved, please merge